static/js/auth.ts

```diff
@@ -362,7 +362,7 @@ $ ('#email, #mail_repeat').on ('cut copy paste', function (e) {
  * - Check if we were supposed to be joining a class. If so, join it.
  * - Otherwise redirect to "my programs".
  */
-async function afterLogin(teacher: boolean) {
+async function afterLogin(loginData: any) {
```

Comment:
I personally really don't care, but, if you want to leverage typescript here, you may want to use type annotations;

```typesccript
loginData: { teacher: boolean }
```

Reply:
@ThomasKluiters You made this remark (which has not been changed) but also approved so now I am not sure if I should await this change or merge?